db/schema.rb

```diff
@@ -29,6 +29,7 @@
     t.bigint "portfolio_item_id"
     t.jsonb "service_parameters"
     t.jsonb "provider_control_parameters"
+    t.string "context"
```

Comment:
@lindgrenj6 Should this be a JSONB column

Reply:
https://github.com/ManageIQ/manageiq-schema/blob/3d5c661ab2684029a76d9c5851a212f1d05724d2/db/migrate/20170519153350_add_showback_models.rb#L16

Reply:
Yes it should be, that is a typo on my part. Fixed.